analyses/cell-type-wilms-tumor-06/README.md

```diff
@@ -0,0 +1,178 @@
+# Wilms Tumor Dataset Annotation (SCPCP000006)
+
+Wilms tumor (WT) is the most common pediatric kidney cancer characterized by an exacerbated intra- and inter- tumor heterogeneity.
+The genetic landscape of WT is very diverse in each of the histological contingents.
+The COG classifies WT patients into two groups: the favorable histology and diffuse anaplasia.
+Each of these groups is composed of the blastemal, epithelial, and stromal populations of cancer cells in different proportions, as well as cells from the normal kidney, mostly kidney epithelial cells, endothelial cells, immune cells and normal stromal cells (fibroblast).
+
+## Description
+
+Here, we first aim to annotate the Wilms Tumor snRNA-seq samples in the SCPCP000006 (n=40) dataset. To do so we will:
+
+• Provide annotations of normal cells composing the kidney, including normal kidney epithelium, endothelium, stroma and immune cells
+• Provide annotations of tumor cell populations that may be present in the WT samples, including blastemal, epithelial, and stromal populations of cancer cells
+Based on the provided annotation, we would like to additionally provide a reference of marker genes for the three cancer cell populations, which is so far lacking for the WT community.
+
+The analysis is/will be divided as the following:
+
+- [x] Metadata file: compilation of a metadata file of marker genes for expected cell types that will be used for validation at a later step
+- [ ] Script: clustering of cells across a set of parameters for few samples
+- [ ] Script: label transfer from the fetal kidney atlas reference using runAzimuth
+- [ ] Script: run InferCNV
+- [ ] Notebook: explore results from steps 2 to 4 for about 5 to 10 samples
+- [ ] Script: compile scripts 2 to 4 in a RMardown file with required adjustements and render it across all samples
+- [ ] Notebook: explore results from step 6, integrate all samples together and annotate the dataset using (i) metadatafile, (ii) CNV information, (iii) label transfer information
+
+## Usage
+From Rstudio, run the Rmd reports or render the R scripts (see below R studio session set up).
+Please before running the script, make sure that the paths are correct.
```

Comment:
I don't know if we need this sentence now that the other instructions use the data download script.

```suggestion
```